compiler/surface/lexer_pl.cppo.ml

```diff
@@ -29,6 +29,7 @@
 #define MS_INCREASING "rosnacy"
 #define MS_OF "z"
 #define MS_COLLECTION "kolekcja"
+#define MS_CONTAINS "contains"
```

Comment:
The translation of `contains` in Polish seems to be "[zawiera](https://context.reverso.net/translation/polish-english/zawiera)"

Reply:
```suggestion
#define MS_CONTAINS "zawiera"
```